docs/start/qiskit-versioning.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Qiskit Versioning and Support Policy
+description: The versioning and support policy for Qiskit.
+
+---
+<span id="qiskit-version"></span>
```

Comment:
You can have this if you want, but we'll already have an anchor link `qiskit-versioning` from the header.